content/advanced/fine-grained-reactivity.mdx

```diff
@@ -236,13 +242,20 @@ createEffect(() => {
 });
 ```
 
-Here, the `createEffect` function initiates a `setTimeout`, which is asynchronous. 
-Because the system is synchronous, it doesn't wait for this asynchronous operation to complete. 
+Here, the `createEffect` function initiates a `setTimeout`, which delays the call. 
+Because the system is synchronous, it doesn't wait for this operation to complete. 
 By the time the `count` getter is invoked within the `setTimeout`, the global scope no longer has a registered subscriber.
 As a result, this `count` signal won't add the callback as a subscriber, leading to potential issues in tracking changes to count.
 
 ### Handling asynchronous effects
 
 While the basic reactivity system is synchronous, frameworks like Solid offer more advanced features to handle asynchronous scenarios. 
 For example, Solid provides the `on` function, which allows for manually specifying dependencies for an effect. 
-This is particularly useful for ensuring that asynchronous operations are correctly tied into the reactive system. 
\ No newline at end of file
+This is particularly useful for ensuring that asynchronous operations are correctly tied into the reactive system. 
+
+In addition to the `on` function, Solid introduces the concept of resources for managing asynchronous operations. 
```

Comment:
I guess one can say that `on` has something to do with asynchronicity, but it seems kinda weird to say it 🤔 like it's just a tool that fixes the list of dependencies, it doesn't really even know that asynchronicity exists.